src/lib/Filters.js

```diff
@@ -163,14 +163,19 @@ export const Constraints = {
     field: null,
     comparable: false
   },
+  dneOrNull: {
+    name: 'does not exists or null',
+    field: null,
+    comparable: false
```

Comment:
If you want to add additional filters you can set comparable to `true` also we should document what `comparable` and `composeable` (I think it allows you to add a value to compareTo) are